MAUI/Carousel-View/how-to.md

```diff
@@ -0,0 +1,126 @@
+---
+layout : post
+title: Interaction in Syncfusion Carousel Control in .NET MAUI.
+description: Learn how to perform an operation while changing the carouselItem or Collection in Carousel for .NET MAUI.
+platform : maui
+control : Carousel
+documentation : ug
+---
+
+# How to perform an operation while changing the carouselItem?
+
+We can perform operation while changing the carouselItem using `SelectionChanged` event. `SelectionChanged` event returns the index and selected carouselItem
```

Comment:
We can perform the operation while changing the carouselItem using the `SelectionChanged` event. The `SelectionChanged` event returns the index and selected carouselItem.

Reply:
Changes Addressed